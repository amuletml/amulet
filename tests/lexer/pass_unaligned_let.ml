let ok = let x = 2 in x

let under = let x = 2
          in x

let over = let x = 2
            in x
