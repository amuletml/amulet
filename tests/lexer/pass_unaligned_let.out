let ok = let x = 2 $end in x

$end $end $sep let under = let x = 2
          $end in x

$end $end $sep let over = let x = 2
            $end in x
$end $end <eof>
(*
  pass_unaligned_let.ml[4:11 ..4:11]: warning
    This `in` is misaligned with the corresponding `let`
    │ 
  3 │ let under = let x = 2
  4 │           in x
    │ 
  pass_unaligned_let.ml[7:13 ..7:13]: warning
    This `in` is misaligned with the corresponding `let`
    │ 
  6 │ let over = let x = 2
  7 │             in x
    │ 
*)
